runtime/rasdump/javadump.cpp

```diff
@@ -1776,6 +1782,54 @@ JavaCoreDumpWriter::writeMemoryCountersSection(void)
 		"NULL\n"
 		"NULL           ------------------------------------------------------------------------\n"
 	);
+#if defined(J9ZTPF)
+	struct immap_privatemstats ztpfNativeStats;
+	int rc;
+
+	if (MAP_FAILED != mmap(NULL, 0, 0, MAP_SUPPORTED, 0, 0)) {
+		rc = mprivatestats(&ztpfNativeStats);
+
+		_OutputStream.writeCharacters("0SECTION         Native configuration information for z/TPF\n");
+		_OutputStream.writeCharacters("NULL             ==========================================\n");
+		if (0 != rc) {
+			_OutputStream.writeCharacters("1TPFCONFIG      Unavailable: ");
+			_OutputStream.writeInteger(rc, "%d\n");
```

Comment:
This is missing casts (my bad advice):
```c
			_OutputStream.writeInteger((UDATA)(IDATA)rc, "%zd\n");
```
Also, each call to `writeInteger()` should use the `z` modifier (`%zu` instead of just `%u`); similar to line 1823.